src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3507,6 +3509,21 @@ void ApplicationManagerImpl::ForbidStreaming(uint32_t app_id) {
     UnregisterApplication(app_id, ABORTED);
     return;
   }
+
+  switch (service_type) {
+    case ServiceType::kMobileNav:
+      if (app->video_streaming_allowed()) {
+        return;
```

Comment:
@SNiukalov suggest to add some logging before these returns just for understanding why function was executed in that way

Reply:
@AKalinich-Luxoft 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3293/commits/cde8652542fc0a0e431ae0a73e877022041d35da